draft-yasskin-http-origin-signed-responses.md

```diff
@@ -838,6 +838,10 @@ signature returns "valid", return "valid". Otherwise, return "invalid".
 1. Let `exchange` be the exchange metadata and headers parsed out of `headers`.
 1. If `exchange`'s request method is not safe (Section 4.2.1 of {{!RFC7231}}) or
    not cacheable (Section 4.2.3 of {{!RFC7231}}), return "invalid".
+1. If `exchange`'s response is not complete, (Section 3.1 of {{!RFC7231}}),
+   return "invalid".
+1. If `exchange`'s response is not cachable by a public cache (Section 3 of
```

Comment:
I believe "shared cache" is the right term, according to https://httpwg.org/specs/rfc7234.html#rfc.section.1. "public" is a response directive, not a type of cache.

I'd also rather not use the term "cacheable" here since https://httpwg.org/specs/rfc7234.html#rfc.section.3 doesn't define it as a predicate on responses. Instead perhaps:

```suggestion
1. If Section 3 of {{RFC7234}} forbids a shared cache from storing `exchange`'s response, return "invalid".
```

Reply:
Done.